### PR TITLE
Catch invalid line breaks exception from CsvReader

### PR DESCRIPTION
**Pull request by webrouse**
**Error in log:** https://my.papertrailapp.com/groups/23635/events?focus=1222931146619367425&q=%22keboola.ex-db-mssql%22%20AND%20%22exception%22%20AND%20%22attachment%22&selected=1222931146619367425

**Slack:** https://keboola.slack.com/archives/C09U3R1J4/p1595506745059500

**Changes:**
- When BCP generates CSV with invalid line breaks, exception is catched and PDO fallback works.
- There were missing `catch` in code.
- I was not able to simulate the error locally.
- It was working in previous version in same way:

image

### CODE REVIEW

File: src/Extractor/Adapters/BcpAdapter.php [hunk 1]
```diff
@@ -233,7 +242,7 @@ private function createBcpCommand(string $filename, string $query): string
         $serverName .= ',' . $this->dbParams['port'];
 
         $cmd = sprintf(
-            'bcp %s queryout %s -S %s -U %s -P %s -d %s -q -k -b50000 -m1 -t, -r"\n" -c',
+            'bcp %s queryout %s -S %s -U %s -P %s -d %s -q -k -b 50000 -m 1 -t "," -r "\n" -c',
```
Comment (webrouse) [inline src/Extractor/Adapters/BcpAdapter.php:245]: Rovno som fixol aj chybajuce medzery medzi argumentami. 
Pri testovani som si vsimol, ze batch size je podla logu 1000 aj ked je tam nastavene 50000.
Ostatne hodnoty su defaultne, takze sa tam chybajuca medzera neprejavila.

Comment (pivnicek) [reply src/Extractor/Adapters/BcpAdapter.php:245]: omg, diky!

**Comment by ujovlado** [comment]
@webrouse ak by si chcel, myslim, ze viem ako nasimulovat tie zle Line breaks lokalne.

Staci ak do db vlozis line breaks `\r` a ides to exportovat bez akehokolvek replace `char(10)` a `char(10)`

**Comment by webrouse** [comment]
@ujovlado vdaka, to dava zmysel ... myslel som si, ze ide o chybu BCP, ... ale pozeral som si teraz v projekte ake data exportuje PDO fallback,... a su tam nejake XMLka asi s Mac zalomením riadkov.

Ono ta detekcia line breaks nie je velmi spolahliva, ... ono by sa to tam muselo prechadzat po znakoch, a brat do uvahy separatory a escapovanie, aby to bolo spolahlive.
https://github.com/keboola/php-csv/blob/86f03def371e5b1a8a11973b496c86d3a0d4d57d/src/CsvReader.php#L112-L137

**Comment by ujovlado** [comment]
To len tak na okraj - ja som si to nahodou vsimol popri debugu ;)

**Comment by webrouse** [comment]
Fix v php-csv: https://github.com/keboola/php-csv/pull/41

**Comment by webrouse** [comment]
Fixol som to v `php-csv`, teda tento PR zatvaram a spravim novy:
https://github.com/keboola/php-csv/pull/41

**Pull request closed without merging**